web_src/js/index.js

```diff
@@ -729,9 +729,45 @@ function initRepository() {
       return false;
     });
 
+    // Issue/PR Context Menus
+    $('.context-dropdown').dropdown({
+      action: 'hide'
+    });
+
+    // Quote reply
+    $('.quote-reply').click(function (event) {
+      $(this).closest('.dropdown').find('.menu').toggle('visible');
+      const target = $(this).data('target');
+
+      let $content;
+      if ($(this).hasClass('quote-reply-diff')) {
+        const $parent = $(this).closest('.comment-code-cloud');
+        $parent.find('button.comment-form-reply').click();
+        $content = $parent.find('[name="content"]');
+      } else {
+        $content = $('#content');
+      }
+
+      let content = $(`#comment-${target}`).text();
+      const parts = content.split('\n');
+      for (let i = 0; i < parts.length; i++) {
+        parts[i] = `> ${parts[i]}`;
```

Comment:
Maybe easier
```
let content = '> ' + $(`#comment-${target}`).text().replace('\n','\n> ') + '\n\n'
```
(or something along those lines)

Reply:
OK, maybe it's not easier, only more compact and fewer steps. 😄

Reply:
Can't concatenate like that, webpack linter gets mad. But I can try to inline it if it's really worth it.

Reply:
> 
> 
> Can't concatenate like that, webpack linter gets mad. But I can try to inline it if it's really worth it.

Nah... Not worth it. I hadn't thought of the impact that something like this could have on a packer, though. Interesting.

Reply:
I did something similar to your suggestion if you would like to look. I just did it in two steps to hopefully make it a bit more readable.